File: SeungMin/황승민:3주차 과제/Q06.c

// 6. 별찍기(하)
// https://www.acmicpc.net/problem/2439

#include <stdio.h>

int main(void) {

    int n;
    scanf("%d", &n);

    for (int i = 1; i <= n; i++ ) {
        for (int j = 1; j <= n-i; j++) {
            printf(" ");
        }
        for (int j = 1; j <= i; j++) {
            printf("*");
        }
        printf("\n");
    }

    return 0;
}

File: SeungMin/Java_week2/Q06.java
package Java_week2;

import java.util.Scanner;

public class Q06 {
    // 크로아티아 알파벳 - https://www.acmicpc.net/problem/2941
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String word = scanner.nextLine() + "aa";
        int cnt = -2;

        for (int i = 0; i < word.length(); i++) {
            if (word.charAt(i) == 'c' && (word.charAt(i+1) == '=' || word.charAt(i+1) == '-')) {
                i++;
            } else if (word.charAt(i) == 'd') {
                if (word.charAt(i+1) == 'z' && word.charAt(i+2) == '=')
                    i += 2;
                else if (word.charAt(i+1) == '-')
                    i++;
            } else if ((word.charAt(i) == 'l' || word.charAt(i) == 'n') && word.charAt(i+1) == 'j')
                i++;
            else if ((word.charAt(i) == 's' || word.charAt(i) == 'z') && word.charAt(i+1) == '=')
                i++;
            cnt++;
        }
        System.out.println(cnt);

    }
}
